bench/BENCHMARK_BASELINE.md

```
@@ -1,513 +1,529 @@
-  Latency      610.29us   324.90us     4.73ms
+  Reqs/sec    168048.22   15979.16  177986.43
+  Latency      583.34us   236.81us     3.83ms
   Latency Distribution
      50%   526.00us
-     75%   712.00us
-     90%     1.00ms
-     99%     2.34ms
+     75%   696.00us
+     90%     0.87ms
+     99%     1.64ms
 ### Redirect Response (/redirect)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)